### PR TITLE
Added Login UI

### PR DESCRIPTION
**Pull request by dieAlarcon**

### CODE REVIEW

File: app/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/vLoginCoverContainer"
+        tools:ignore="ContentDescription" />
+
+    <EditText
+        android:id="@+id/vUsernameInput"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/spacing_large"
```
Comment (DemianIac) [inline app/src/main/res/layout/fragment_login.xml]: If you are assigning the margin with the constraint layout design tool there is no need yo use a resource dimens (just for margin)

Comment (DemianIac) [reply app/src/main/res/layout/fragment_login.xml]: It's still okay here

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -13,4 +13,11 @@
     <string name="fragment_example_username">Username</string>
     <string name="fragment_example_login">Login</string>
 
+    <string name="fragment_login_username">First Name (recommended)</string>
+    <string name="fragment_login_lastname">Last Name (recommended)</string>
+    <string name="fragment_login_login">Log in</string>
+    <string name="fragment_login_signup">Sign up</string>
+    <string name="fragment_login_terms_and_conditions">Terms &amp; Conditions</string>
+    <string name="fragment_login_terms_and_conditions_text">By joining you agree to our</string>
```
Comment (DemianIac) [inline app/src/main/res/values/strings.xml:21]: This should open the actual web page. Is this implemented in other branch?

**Review by DemianIac — COMMENTED** [review]
Nice work! Just a few questions and observations.

When creating a PR you can add this template to the description

### Summary
<Small description of the work>
### Screenshot
<screen of the view>
### Jira Card
<Jira or Trello link>

File: app/src/main/res/layout/fragment_login.xml (hunk 1 above)
Comment (dieAlarcon) [reply app/src/main/res/layout/fragment_login.xml]: Ok, I changed it. Thanks.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (dieAlarcon) [reply app/src/main/res/values/strings.xml:21]: Yes, it's implemented in the next PR.

**Review by DemianIac — APPROVED** [review]
Niceee. Remember to hit the merge button and delete de branch (The button appears when you merge the branch 😛 )

**Pull request merged**